src/Workspaces/SharedUtilitiesAndExtensions/Workspace/VisualBasic/Extensions/ContextQuery/SyntaxTreeExtensions.vb

```diff
@@ -24,7 +24,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
                 End If
 
                 ' If this token is to our left, return it
-                If Not token.IsKind(SyntaxKind.EmptyToken) AndAlso token.Span.End < position Then
```

Comment:
The start of the parenthesized expression for a possible tuple will be the same as the end of the `If` token when there is no space. However, we won't to break the loop at this point. See also line 22 which already have the correct check (`<=`).